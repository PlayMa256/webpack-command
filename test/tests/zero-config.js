const execa = require('execa');
const strip = require('strip-ansi');

const { test } = require('../util');

test('Zero Config', module, () => {
  it('should run', () => {
    const cliPath = resolve(__dirname, '../../lib/cli');
    const cwd = resolve(__dirname, '../fixtures/zero-config');
    const result = execa.sync('node', [cliPath], { cwd });

    expect(
      strip(result.stdout).replace(/Î”t \d+ms/g, '<duration>')
    ).toMatchSnapshot();
  });
});
