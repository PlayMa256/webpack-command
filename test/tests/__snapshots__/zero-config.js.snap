// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Zero Config > should run #0 1`] = `
"ℹ ｢config｣: Config not found, trying webpack.config.es6
ℹ ｢config｣: Config not found, trying webpack.config.flow
ℹ ｢config｣: Config not found, trying webpack.config.mjs
ℹ ｢config｣: Config not found, trying webpack.config.ts
ℹ ｢webpack｣: Starting Build
ℹ ｢webpack｣: Build Finished

webpack v4.3.0

0ca52704a1563f34b0b3
  size   name  module          status
  43 B   0     ./src/index.js  built

  size   name  asset           status
  573 B  main  main.js         emitted

  <duration> 


configuration
  0:0  warning  The 'mode' option has not been set. Set 'mode' option to 'development'
                or 'production' to enable defaults for this environment.

⚠  1 problem (0 errors, 1 warning)"
`;

exports[`Zero Config > should run #1 1`] = `
"ℹ ｢config｣: Config not found, trying webpack.config.es6
ℹ ｢config｣: Config not found, trying webpack.config.flow
ℹ ｢config｣: Config not found, trying webpack.config.mjs
ℹ ｢config｣: Config not found, trying webpack.config.ts
ℹ ｢webpack｣: Starting Build
ℹ ｢webpack｣: Build Finished

webpack v4.3.0

0ca52704a1563f34b0b3
  size   name  module          status
  43 B   0     ./src/index.js  built

  size   name  asset           status
  573 B  main  main.js         emitted

  <duration> 


configuration
  0:0  warning  The 'mode' option has not been set. Set 'mode' option to 'development'
                or 'production' to enable defaults for this environment.

⚠  1 problem (0 errors, 1 warning)"
`;
